src/environment_setup/tool_providers.ml

```diff
@@ -409,7 +411,8 @@ let fastqc =
         shf "cp -r * %s" path
         && shf "chmod +x %s" (binary_path path)
       )
-  ~init_program
+    ~init_program
+    ~unarchived_directory:"FastQC"
```

Comment:
The default value for that is `<name-of-the-tool>*`:
https://github.com/hammerlab/biokepi/blob/62e810311228decd51ab5bb0ef0f5e2638262828/src/environment_setup/tool_providers.ml#L26

And fastqc's zip file seems to unzip to a `FastQC/` directory.

So without the option it would work on MacOSX (“case aware” craziness) but not on a normal filesystem.